index.js

```diff
@@ -59,14 +59,58 @@ animalToRemove.remove()
 /************************** EVENTS JS MINI CHALLENGE ******************************/
 
 /***** Deliverable 1 *****/
+
 function toggleColor(element) {
     if (element.style.color === "green") {
-        element.style.color = "black"
+        element.style.color = "red"
     } else {
         element.style.color = "green"
     }
 }
 
+header.addEventListener('click', function(){
```

Comment:
Good, but the callback function we can pass in to the event listener is already made for us!. We just need to pass in the event target as an argument to the function.
```
header.addEventListener("click", function(e){
    toggleColor(e.target)
})
```
With ES6 notation
```
header.addEventListener("click", e => toggleColor(e.target))
```